openedx/openedx-learning
# feat: Implement ObjectTag retrieve REST API

## Description

This PR implements the tagging REST API for retrieving object tags.

Adding/updating object tags will be handled in a subsequent PR.

## Supporting Information

- Closes https://github.com/openedx/modular-learning/issues/81

## Testing instructions

- Ensure that the tests cover the expected behavior of the view as described in the related issue.

---
Private-ref: [FAL-3452](https://tasks.opencraft.com/browse/FAL-3452)
